CHANGELOG.rst

```diff
@@ -74,6 +74,8 @@ in development
   When this flag is provided, all triggers contained within a pack triggers directory are
   registered, consistent with the behavior of sensors, actions, etc. This feature allows users
   to register trigger types outside the scope of the sensors. (new-feature) [Cody A. Ray]
+* Allow user scoped variables to be used in action execution run from CLI or UI.
```

Comment:
This really is an API feature, right?

If so, probably a good idea to re-organize this line to make it clear so. I'm also fine with emphasizing that this means it's also supported in CLI and UI as they are one of consumers of the API.